roles/secretgen_controller/tasks/main.yml

```diff
@@ -0,0 +1,921 @@
+# Copyright (c) 2022 VEXXHOST, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Deploy service
+  kubernetes.core.k8s:
+    state: present
+    definition:
+      # Note(Oleks): https://github.com/carvel-dev/secretgen-controller/releases/download/v0.16.0/release.yml
```

Comment:
Can you move the contents of this into `files/release.yml` and then include that into here instead to make this file cleaner?